doc/acknowledgments/CMakeLists.txt

```diff
@@ -16,8 +16,8 @@ SET( INCLUDED_TEX
 SET( INCLUDED_IMAGES
 )
 
-SET( SOURCE_FILENAME "acknowledgements" )
-SET( OUTPUT_FILENAME "Acknowledgements" )
+SET( SOURCE_FILENAME "acknowledgments" )
+SET( OUTPUT_FILENAME "Acknowledgments" )
```

Comment:
I'm ok with fixing the spelling of the acknowledgments document name, but this will also require changes to the link in the [readme](https://github.com/NREL/EnergyPlus/blob/254f35681ee9ae1b80711faccca7049e8847c5c2/release/readme.html#L122) and a companion issue to change the Help menu links in EP-Launch and IDF Editor.  @jasondegraw has suggested eliminating this document altogether, but I don't know when that might happen.

Reply:
OK. I'll go ahead and create a separate issue for the other changes.